agents/notion/README.md

```diff
@@ -0,0 +1,32 @@
+# Fixie Notion Agent
```

Comment:
I feel like this is not a good Notion example because it is highly specialized on the data format within Notion. Our example should work for any customer's existing Notion site.

If you still want this to go into examples in its current form we should change the name to avoid making it seem like this is a general Notion agent.